### PR TITLE
Pull request for create-gantt entry (Issue #4563)

### LINKED ISSUE #4563: [Term Entry] Python Plotly .create_gantt()
### Reason/inspiration (optional)

We would like a new term entry in the `Data Visualization` concept for Plotly: `.create_gantt()`. The entry should go in a new file under `docs/content/plotly/concepts/figure-factory/terms/create-gantt/create-gantt.md`

The entry should describe the term and how it is used. Also include:

- An introductory paragraph that describes the concept
- A 'Syntax' section that includes the syntax for the concept (if applicable)
- An 'Example' section that shows the concept in use

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by rgo444**
### Description

New pull request created for the create-gantt method of the plotly library. One associated image included in media, regarding the output of the example code included in the entry.

### Issue Solved

Closes #4563 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by mohitO12-dev — APPROVED** [review]

**Review by mohitO12-dev — APPROVED** [review]

**Review by mohitO12-dev — APPROVED** [review]

File: content/plotly/content/figure_factory/terms/create-gantt/create-gantt.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+```
+
+The `.create_gantt` method takes the following arguments:
+
+- `df`: Required input data, either a `data frame` or a list, serving as the foundation for the Gantt chart's content.
+- `colors`: Specifies the color scheme used in the chart, allowing customization through options like a Plotly scale name, a `list` of colors, a `dictionary` mapping values to colors, or a `tuple` defining the color range.
+- `index_col`: Identifies the column header to be used as the indexing column in the Gantt chart, aiding in the organization of tasks or data points along the `y-axis`.
+- `show_colorbar`: A `boolean` parameter determining the visibility of the color bar in the Gantt chart.
+- `reverse_colors`: A `boolean` parameter enabling the reversal of selected colors in the chart.
+- `title`: Sets a descriptive title for the Gantt chart.
```
Comment (cigar-galaxy82) [inline content/plotly/content/figure_factory/terms/create-gantt/create-gantt.md:36]:
```suggestion
- `df`: Required input data, either a `data frame` or a list, serving as the foundation for the Gantt chart's content.
- `colors`: Specifies the color scheme used in the chart, allowing customization through options like a Plotly scale name, a list of colors, a dictionary mapping values to colors, or a tuple defining the color range.
- `index_col`: Identifies the column header to be used as the indexing column in the Gantt chart, aiding in the organization of tasks or data points along the y-axis.
- `show_colorbar`: A boolean parameter determining the visibility of the color bar in the Gantt chart.
- `reverse_colors`: A boolean parameter enabling the reversal of selected colors in the chart.
- `title`: Sets a descriptive title for the Gantt chart.
```

**Comment by cigar-galaxy82** [comment]
Hey @rgo444

Thank you for contributing to Codecademy Docs! The entry is well written!

I've suggested a few changes modify them at your convenience

Thank you!

**Review by mohitO12-dev — APPROVED** [review]
all fixes completed and approved.

**Review by mohitO12-dev — APPROVED** [review]
after discussion with the @cigar-galaxy82 and my all error is fixed this PR is going on merge.

**Pull request merged**